docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+# A docker-compose to make it easier to run PHP integration tests locally with different PHP versions.
```

Comment:
As I was wrestling with inconsistent CI build failures, I ended up creating a docker-compose to make it easier to test ideas locally. Thought it might help others in the future.